certified-connectors/YakChat/apiDefinition.swagger.json

```diff
@@ -1,397 +1,945 @@
 {
-  "swagger":"2.0",

```

Comment:
@aMatthewsYakChat may I ask you please to fix the file format https://github.com/microsoft/PowerPlatformConnectors/pull/2455/files#diff-db9a1bbbeb21aff8cc57defd4c38d5a18917d471abe7a71c342e9310eb01d8f1. It looks like you deleted everything and readded it again. 
Please feel free to close this PR and create a new one after you update your branch with our dev branch.
![image](https://user-images.githubusercontent.com/97551789/223784942-d8710772-606e-497e-9420-b4bc35d048c7.png)

Reply:
I have updated the branch with dev, but can I ask for some clarification on what is incorrect with the file format?

Reply:
@aMatthewsYakChat Please make sure that you have the correct swagger in [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/YakChat/apiDefinition.swagger.json). before you submit to ISV Studio.